mesheryctl/internal/cli/root/pattern/apply.go

```diff
@@ -117,7 +116,8 @@ mesheryctl pattern apply [pattern-name]
 			}
 		} else {
 			// Method to check if the entered file is a URL or not
-			if validURL := govalidator.IsURL(file); !validURL {
+			validURL := strings.HasPrefix(file, "https://github.com")
```

Comment:
We’re not mandating that URLs contain github.com, are we?

Reply:
This line calls ParseUrlGithub and i saw the function it works for host github.com and raw.githubusercontent.com
https://github.com/meshery/meshery/blob/ad9b22e3f32a13b30fc7f5a120be075ab22d324f/mesheryctl/internal/cli/root/pattern/apply.go#L170

For Reference
https://github.com/meshery/meshery/blob/ad9b22e3f32a13b30fc7f5a120be075ab22d324f/mesheryctl/pkg/utils/helpers.go#L573